### DISCUSSION TITLE
Normal Structured Data Regression

### DISCUSSION THREAD

**Discussion by Jiaxin8122** [keras-team/autokeras#1453] [Q&A]
Hi,

I'm trying to find some toturials online with StructuredDataRegressor. When i use this command "search.fit(x=X_train, y=y_train, verbose=0)“, some errors reported...
TypeError: __init__() got an unexpected keyword argument 'verbose'
Also when i set "time_limit=60*60" in this command, it said, no attribute“time_limit”, i just copy code from online. I use autokeras 1.0.11. Is the version issue? Because i saw tutorials online they use 0.2.11.  Can someone help me? I'm totally fresh to autokeras.

Regards

**Comment by haifeng-jin** [comment]
After 1.0, many args has changed.
